### ISSUE TITLE
[ts] Response code 0 in file system

### ISSUE THREAD

**Issue by edbentley** [EsotericSoftware/spine-runtimes#1969]
I'm loading  Spine in a WKWebView on iOS. When I load files I get the error:

```
Error: Asset not found: skeleton.atlas
Couldn't load text skeleton.atlas: status 0
```

I found in the source code [here](https://github.com/EsotericSoftware/spine-runtimes/blob/4.0/spine-ts/spine-core/src/AssetManagerBase.ts#L311) and [here](https://github.com/EsotericSoftware/spine-runtimes/blob/4.0/spine-ts/spine-core/src/AssetManagerBase.ts#L333) that you're expecting a status code of `200`. However on local file systems the response code can be `0` and so is throwing an error when there isn't one.

I'm using `"@esotericsoftware/spine-canvas": "4.0.14"`.

**Comment by badlogic** [comment]
I'm not familiar with local file system access inside WKWebView. Do you have an example I could use to debug, fix, and verify this with?

**Comment by edbentley** [comment]
@badlogic It isn't specific to WKWebView. I can reproduce using the [canvas-example](https://github.com/EsotericSoftware/spine-runtimes/tree/4.0/spine-ts/spine-canvas/example) when I open Chrome with security disabled (just for testing):

```
chrome --disable-web-security --user-data-dir="/some-scrap-directory"
open index.html
```

**Comment by badlogic** [comment]
This is now fixed in the 4.0 and 4.1-beta branches, and released as version 4.0.16 and 4.1.4 on NPM.
